addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -561,24 +560,25 @@
 
                 <a t-if="slide_promoted.is_preview or channel.is_member or is_slides_publisher"
                 t-attf-href="/slides/slide/#{slug(slide_promoted)}#{query_string}" class="w-100 w-lg-50 flex-shrink-0 rounded">
-                    <div t-attf-style="background-image:url(#{image_url}); background-size: cover; background-position:center; padding-bottom: 35%;" class="h-lg-100"/>
+                    <div t-if="slide_promoted.image_1920" t-field="slide_promoted.image_1920" t-options='{"widget": "image", "style":"height:100%"}' class="h-100"/>
```

Comment:
Last comment :smile:  We should probably add in the `t-options` a `"preview_image": "image_1024"` when we use a `t-field` for performance. I see that we used mainly `image_1024` previously but we could probably also use a lower one in some cases like in `lesson_card` with `image_512` perhaps. What do you think ?